Sources/TSCBasic/Process.swift

```diff
@@ -324,9 +325,28 @@ public final class Process: ObjectIdentifierProtocol {
                 pathString: ProcessEnv.path,
                 currentWorkingDirectory: localFileSystem.currentWorkingDirectory
             )
+#if os(Windows)
+            var searchPaths = Array<AbsolutePath>()
+            var buffer = Array<WCHAR>(repeating: 0, count: Int(MAX_PATH + 1))
+
+            // The 32-bit Windows system directory
+            GetSystemDirectoryW(&buffer, .init(MAX_PATH + 1))
+            searchPaths.append(AbsolutePath(String(decodingCString: buffer, as: UTF16.self)))
+
+            // The 16-bit Windows system directory
+            searchPaths.append(AbsolutePath("\(ProcessEnv.vars["systemdrive"] ?? "C:")\\System"))
+
+            // The Windows directory
+            GetWindowsDirectoryW(&buffer, .init(MAX_PATH + 1))
+            searchPaths.append(AbsolutePath(String(decodingCString: buffer, as: UTF16.self)))
+
+            searchPaths.append(contentsOf: envSearchPaths)
```

Comment:
Why these specific directories?  Why not the traditional lookup via Path?  What happens when there's a shadowing g variant that is elsewhere in Path and you invoke the wrong executable?  I think that `SearchPathW` would be better.

Reply:
I got the order from https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessw . It looks like `%PATH%` has the lowest priority.

Reply:
Correct, the path is added to a list of additional search paths.  That doesn't mean that we shouldn't honor them.